uaclient/util.py

```diff
@@ -576,3 +577,18 @@ def write_file(filename: str, content: str, mode: int = 0o644) -> None:
         fh.write(content.encode("utf-8"))
         fh.flush()
     os.chmod(filename, mode)
+
+
+def get_allow_beta_value_from_config(config_dict):
```

Comment:
Sorry, near "bike shedding" here  lets make this utility more general and less specific to allow_beta.

Can we make this function just is_config_true(key_path:str, value: str):
...
raise exceptions.UserFacingError(status.ERROR_INVALID_CONFIG_VALUE.format(key_path=key_path, expected_value="boolean string: true or false", value=value

And we just make the call site:
```
allow_beta = config_dict.get("features", {}).get("allow_beta", "false")
util.is_config_true(key_path="features.allow_beta", value=allow_beta)
```